src/cgame/default/Makefile.am

```diff
@@ -66,5 +66,6 @@ cgame_la_LDFLAGS = \
 cgame_la_LIBADD = \
 	viewcontrollers/libviewcontrollers.la \
 	$(top_builddir)/src/game/default/libpmove.la \
-	$(top_builddir)/src/libswap.la
+	$(top_builddir)/src/libswap.la \
+	$(top_builddir)/src/libmatrix.la
```

Comment:
I'm okay with adding libmatrix.la as a dependency for cgame. We do want to be careful, in general, with introducing deps to cgame / game, but this one probably makes sense.